frontend/src/assets/convert.sh

```
@@ -0,0 +1,54 @@
+# brew install imagemagick
+#!/bin/bash
```

Shellcheck (0.10.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)